sources/packages/workflow/BPMN/camunda-8/load-assessment-consolidated-data.bpmn

```diff
@@ -10,32 +10,32 @@
         <zeebe:taskDefinition type="load-assessment-consolidated-data" />
         <zeebe:taskHeaders>
           <zeebe:header key="assessmentTriggerType" value="$.triggerType" />
-          <zeebe:header key="isYourSpouseACanadianCitizen" value="$.data.isYourSpouseACanadianCitizen" />
-          <zeebe:header key="doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber" value="$.data.doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber" />
-          <zeebe:header key="numberOfParents" value="$.data.numberOfParents" />
-          <zeebe:header key="dependantstatus" value="$.data.dependantstatus" />
+          <zeebe:header key="studentDataIsYourSpouseACanadianCitizen" value="$.data.isYourSpouseACanadianCitizen" />
```

Comment:
Like this way of organizing the variables 👍

Reply:
@abschwenker works 😉